oracle/oracle-explanation/canonical-offerings.rst

```diff
@@ -0,0 +1,57 @@
+
+* Boot firmware - boot with BIOS or UEFI (preferred). ARM64 images only support UEFI boot.
+* Launch modes - support for both PARAVIRTUALIZED and NATIVE launch modes.  
+* Network interfaces - support for VFIO and PARAVIRTUALIZED modes.
+* Boot volume -  allows booting through iSCSI or PARAVIRTUALIZED storage.
+
+The customisations include NVMe tunings, iSCSI configurations to allow instances to boot from iSCSI backend when appropriate, logging improvements, iptables and udev configurations specific to Oracle Cloud platform, among others.
+
+
+Kernel optimisations
+--------------------
+
+Ubuntu images available on Oracle Cloud run the linux-oracle kernel. Additional kernel variants are also offered by Canonical, that users can install as appropriate.
+
+By default Ubuntu images use a **rolling kernel model**, which provides the latest upstream bug fixes and performance improvements around task scheduling, I/O scheduling, networking, hypervisor guests and containers. A rolling kernel model transitions the default linux-oracle kernel from one base version to the next as part of its regular patching cycle. That new kernel, called the HWE Kernel (Hardware Enablement Kernel) is the kernel of the latest Ubuntu release.
+
+As an example, users running Ubuntu 22.04 instances that were launched back in 2022, would run the 5.15 kernel by default. But as new Ubuntu versions got released, the linux-oracle kernel rolled on to the next release's kernel version. So an Ubuntu 22.04 instance launched at a later point in time, would include different kernel versions depending on when it was launched - it could be version 6.5 or and more recently 6.8.
+
+If you do not want to roll to a new kernel, and want to stay on the base kernel provided by the LTS release (which continues to get support and receive updates for the length of the LTS), you need to install a specific corresponding kernel variant: `linux-oracle-lts-<release>`. (Refer to the next section for an example.)
+
+
+Kernel variants
+~~~~~~~~~~~~~~~
+
+Canonical provides different kernel variants, all optimised for Oracle Cloud: 
+
+* linux-oracle: The default rolling kernel
+* linux-oracle-edge: The -edge kernel provides early access to the next HWE kernel. It is fully supported, but is less exposed to real world use cases since it is relatively new. It eventually transitions to the linux-oracle kernel.
+* linux-oracle-lts-<release>: This kernel does not roll and sticks to the original kernel present in the Ubuntu release, for the life of the release (e.g.: linux-oracle-lts-22.04 will always point to a 5.15 kernel for the life of Ubuntu 22.04).
+* linux-oracle-64k: This is a kernel specific to ARM64 instances, and uses 64k pages by default (as opposed to 4k pages in other kernels). It is known to improve performance in large ARM64 based instances, but is not meant for general use cases, as it can cause issues with low memory instance types.
```

Comment:
It might be better to change "large" in this sentence to something more specific -- high-memory, or many-cpus, or something else, as the term "large" is ambiguous

Reply:
@fabiomirmar could you help with this too? Thanks!

Reply:
Agreed. I think it makes sense to change from "large" to "high memory", since that's what matters for the 64k page choice.

Reply:
Updated to 'high memory'.